### ISSUE TITLE
[Question] How to resume/update a plot across runs

### ISSUE THREAD

**Issue by skrish13** [facebookresearch/visdom#109]
Example: Assume I'm plotting the loss of a neural network. How to update the plot when I'm re-running the code (resuming it from when I stopped).

**Comment by ajabri** [comment]
Hi @skrish13 

Thanks for your message. When you create a pane, you should get the id as a return value. You can also supply a pane-id as a target for your plot. So, you can do something like

```
id = None
... train ...
id = vis.line(X=mydata, win=id)
```

**Comment by skrish13** [comment]
Right, this is cool. So basically, I should somehow store those pane-ids so that I can use it across the runs right?

**Comment by ajabri** [comment]
Yep, or you can use your own id:  `vis.line(X=mydata, win='wazzup')`

**Comment by skrish13** [comment]
ohhh. I kinda assumed you first have to create a pane to get the id. This is epic. Thanks :D
